README.md

```diff
@@ -126,6 +126,14 @@ follows.
 | `g:cmake_link_compile_commands` | `0`                |
 | `g:cmake_root_markers`          | `['.git', '.svn']` |
 
+Vim-CMake provides a set of custom events to trigger further actions.
```

Comment:
Could we move this paragraph just above the "### Quickfix list" section, and make it into an "### Events" section, to match the order in doc/cmake.txt?